### DISCUSSION TITLE
This looks interesting, but the documentation seems to be incomplete

### DISCUSSION THREAD

**Discussion by jesse-stewart** [tsirysndr/music-player#177] [General]
Hello - this looks interesting, but I noticed the documentation is missing some information.

- Where do I put audio files?
- how do I start the tui?

Thank you

**Comment by jesse-stewart** [comment]
I see in the -help that scan will be looking for a /Music directory.

Maybe it is something I am doing wrong, but when running music-player scan I do not see any output in the console.

**Reply by tsirysndr** [reply]
$HOME/Music is default music directory, can be overridden at `$HOME/.config/music-player/settings.toml`, may be your Music directory doesn't contain any supported media files? (mp3, m4a ...
![Screenshot 2024-05-09 8 47 56 PM](https://github.com/tsirysndr/music-player/assets/15877106/229a1707-33b1-49f8-90e5-03532aac8ddc)
)
